pkg/app/mutate_app.go

```diff
@@ -246,9 +248,16 @@ func (m *Mutator) mutateKubeConfig(ctx context.Context, app v1alpha1.App) ([]mut
 		return nil, nil
 	}
 
+	contextName := app.Namespace
```

Comment:
I guess this is for backwards compatibility?

Reply:
That's right, until we fully migrate we still need to consider apps in cluster namespace